apps/api/src/app/controllers/__tests__/__snapshots__/fetchManyViewModels.e2e.spec.ts.snap

```diff
@@ -44,6 +44,34 @@ Array [
 ]
 `;
 
+exports[`When fetching multiple entities GET /transcribedAudioItems when all of the entities are published should fetch multiple entities of type audioWithTranscript 1`] = `
+Array [
+  Object {
+    "audioURL": "https://www.myaudio.com:5555/media?id=123.mp3.mp3",
+    "id": "110",
+    "length": 20000,
+    "plainText": "
+[12000] There once was a little wooden boy. [15550]
```

Comment:
It would be interesting to explore this more and compare to the different subtitle file formats to see how we'd convert.

Reply:
We discussed this, but we can get back to it later.